member-star.scm

```diff
@@ -0,0 +1,13 @@
+(define member*
+  (lambda (a l)
+    (cond
+      ((null? l) #f)
+      ((atom? (car l))
+        (or (eq? (car l) a)
+          (member* a (cdr l))))
+          (else (or (member* a (car l))
```

Comment:
the indent should be 6 spaces

Reply:
ok thank you i will check it later.you work is appreciated